### DISCUSSION TITLE
单独使用MSFU和hardsource-webpack-plugin会冲突吗

### DISCUSSION THREAD

**Discussion by jianxing-xu** [umijs/umi#12493] [Q&A]
单独使用MSFU和hardsource-webpack-plugin会冲突吗，有大佬大nuxt2项目中使用过MFSU吗

**Comment by fz6m** [comment]
hard source webpack plugin 是 webpack 4 时代的东西，已经很多年不维护并被放弃了，所以请不要在 webpack 5 的项目中使用此插件。

**Reply by jianxing-xu** [reply]
> hard source webpack plugin 是 webpack 4 时代的东西，已经很多年不维护并被放弃了，所以请不要在 webpack 5 的项目中使用此插件。

感谢回复，因为我在Umi文档中看到了单独使用 MFSU webpack4 的例子，正好我们的项目是nuxt2，想知道nuxt中的nuxt.config.js怎么和MFSU结合，所以想问一下，不知道团队有没有在nuxt2中做过测试，非常感谢！

**Reply by fz6m** [reply]
nuxt 是 SSR ，不适合使用 MFSU ，nuxt 2 已经 EOL 不维护了，建议升级到 nuxt 3 或 4 。
